microcosm_daemon/tests/test_runner.py

```diff
@@ -63,3 +72,20 @@ def test_process_runner_to_terminate_pool(signum):
 
     pool.close.assert_called_once()
     pool.terminate.assert_called_once()
+
+
+if __name__ == "__main__":
+    daemon = FixtureDaemon()
+    daemon.run()
+
+
+def test_healtcheck_server():
+    popen_instance = Popen(
+        "python microcosm_daemon/tests/test_runner.py --processes 2 --heartbeat-threshold-seconds 2",
+        shell=True,
```

Comment:
This ended up being the simplest approach to run the daemon with multiple processes in tests.
Thank you @dumpstate for helping me debug the many weirdnesses in earlier iterations of this test.